tests/test_bsfs.c

```diff
@@ -203,6 +203,14 @@ START_TEST(test_mknod) {
 }
 END_TEST
 
+START_TEST(test_mknod_file_exists) {
+  char path[256];
+  strcpy(path, mknod_level_name);
+  ck_assert_int_eq(bsfs_mknod(tmp_fs, strcat(path, "/bla"), S_IFREG), 0);
```

Comment:
Maybe the `strcat` should be moved up to a separate line. This makes it less confusing and more consistent with the rest of the tests.

Reply:
I thought about that but decided it wasn't really worth the effort since no one is really going to go over these tests but if it really is an issue I will change it